myoolala/react-starter
# Feat/fargate example

## Acceptance Criteria
Deploying the base app creates a stable fargate service

## Change Description

- Deploying the base app creates a stable fargate service
- Updated terraform and terragrunt to the latest version
- Updated node to v18
- Updated the git ignore to allow for better testing
- Fixed the production node docker file to work without volumes

## Verification Instructions
- Run the local build of the prod node docker file without any volumes and confirm the app works
- Fill in the needed info for the fargate terragrunt file and deploy it by deploying just the ecr repo and then using the deploy script

## Commit Message
- feat: Deploying the base app creates a stable fargate service
- chore: Updated terraform and terragrunt to the latest version
- chore: Updated node to v18
- feat: Updated the git ignore to allow for better testing
- bugfix: Fixed the production node docker file to work without volumes
